draft-ietf-dhc-rfc8415bis-01.xml

```diff
@@ -1789,11 +1790,6 @@
       contain unknown options (or instances of vendor options with unknown
       enterprise-number values). These should be ignored as if they were not
       present.  This is critical to provide for future extensions of DHCP.</t>
-      <t>A server MUST discard any Solicit, Confirm, or Rebind messages
```

Comment:
Do we want to remove the discard of Solicit, Confirm, and Rebind which should be Multicast?